client/src/reader/components/control-menu/AppearanceMenuBody.js

```diff
@@ -68,37 +73,48 @@ class AppearanceMenuBody extends Component {
   };
 
   resetOptionMessage(appearanceType, option) {
-    return `Reader ${appearanceType} is set to ${option}.`;
+    return this.props.t("reader.reset_option_message", {
+      appearanceType,
+      option
+    });
   }
 
   get incrementMarginsMessage() {
-    const baseMessage = "Reader margins increased";
-    const appendedMessage = this.marginIncreaseable ? "." : " to maximum size.";
+    const baseMessage = this.props.t("reader.margin_increased");
+    const appendedMessage = this.marginIncreaseable
+      ? "."
+      : " " + this.props.t("reader.to_max");
     return baseMessage + appendedMessage;
   }
 
   get decrementMarginsMessage() {
-    const baseMessage = "Reader margins decreased";
-    const appendedMessage = this.marginDecreasable ? "." : " to minimum size.";
+    const baseMessage = this.props.t("reader.margin_decreased");
+    const appendedMessage = this.marginDecreasable
+      ? "."
+      : " " + this.props.t("reader.to_min");
     return baseMessage + appendedMessage;
   }
 
   get incrementFontMessage() {
-    const baseMessage = "Reader font size increased";
+    const baseMessage = this.props.t("reader.font_increased");
     const appendedMessage =
-      this.fontSize.current < this.fontSize.max ? "." : " to maximum size.";
+      this.fontSize.current < this.fontSize.max
+        ? "."
+        : " " + this.props.t("reader.to_max");
     return baseMessage + appendedMessage;
   }
 
   get decrementFontMessage() {
-    const baseMessage = "Reader font size decreased";
+    const baseMessage = this.props.t("reader.font_decreased");
     const appendedMessage =
-      this.fontSize.current > this.fontSize.min ? "." : " to minimum size.";
+      this.fontSize.current > this.fontSize.min
+        ? "."
+        : " " + this.props.t("reader.to_min");
```

Comment:
I know it seems like adding duplicates to the translation file, but we shouldn't use partial sentence translations with conditionals. I ran into some like this in `global` that I talked through with Dana the other day. These can be tricky!

I'd probably suggest having two translation keys for each message, each of which returns a full sentence, and then the conditional decides between keys rather than adds a phrase/punctuation. Maybe something like...

```
"font_size_decreased": "Reader font size decreased."
"font_size_decreased_max": "Reader font size decreased to max."

get decrementFontMessage() {
    return this.fontSize.current > this.fontSize.min ? this.props.t("reader.font_size_decreased") : this.props.t("reader.font_size_decreased_max");
}
```